site/profile/manifests/efa.pp

```diff
@@ -0,0 +1,41 @@
+class profile::efa {
+  package { 'libibverbs-utils':
+    ensure => 'installed',
+  }
+
+  package { 'rdma-core-devel':
+    ensure => 'installed',
+  }
+
+  package { 'librdmacm-utils':
+    ensure => 'installed',
+  }
+
+  exec { 'download-efa-driver':
+    command => 'curl -O https://efa-installer.amazonaws.com/aws-efa-installer-1.30.0.tar.gz && tar -xf aws-efa-installer-1.30.0.tar.gz && cd aws-efa-installer',
```

Comment:
They also have `https://efa-installer.amazonaws.com/aws-efa-installer-latest.tar.gz`, their expectation is that you are always using the latest.